guides/plugins/plugins/content/seo/add-custom-seo-url.md

```diff
@@ -0,0 +1,174 @@
+# Add custom SEO URL
+
+## Overview
+
+Every good website had to deal with it already: SEO URLs.
+Of course Shopware supports the usage of SEO URLs, e.g. for products or categories.
+
+This guide however will cover the question on how you can define your own SEO URLs, e.g. for your own
+custom entities.
+This will include both static SEO URLs, as well as dynamic SEO URLs.
+
+## Prerequisites
+
+As every almost every guide in the plugins section, this guide as well is built upon the plugin base guide.
+
+{% page-ref page="../../plugin-base-guide.md" %}
+
+Furthermore, we're going to use a [custom storefront controller](../../storefront/add-custom-controller.md) for the static SEO URL example,
+as well as [custom entities](../../framework/data-handling/add-custom-complex-data.md) for the dynamic SEO URLs.
+Make sure you know and understand those two as well before diving deeper into this guide.
+Those come with two different solutions:
+- Using [plugin migrations](../../plugin-fundamentals/database-migrations.md) for static SEO URLs
+- Using [DAL events](../../framework/data-handling/using-database-events.md) to react on entity changes and therefore generating a dynamic SEO URL
+
+## Custom SEO URLs
+
+As already mentioned in the overview, this guide will divided into two parts: Static and dynamic SEO URLs.
+
+### Static SEO URLs
+
+A static SEO URL doesn't have to change every now and then.
+Imagine a custom controller, which is accessible via the link `yourShop.com/example`.
+
+Now if you want this URL to be translatable, you'll have to add a custom SEO URL to your controller route,
+so it is accessible using both `Example-Page` in english, as well as e.g. `Beispiel-Seite` in german.
```

Comment:
english -> English
germain -> German

Reply:
You sure about that? I know it's the case for the countries, as in "Germany" - but also for the language?